packages/neuron-ui/src/services/localCache.ts

```diff
@@ -138,3 +139,16 @@ export const firstLoadApp = {
     return window.localStorage.getItem(LocalCacheKey.FirstLoadApp) !== 'false'
   },
 }
+
+export const importedWalletDialogShown = {
+  getKey: (walletId: string) => `${walletId}_${LocalCacheKey.ImportedWallet}`,
+  init: (walletId: string) => {
+    window.localStorage.setItem(importedWalletDialogShown.getKey(walletId), 'false')
+  },
+  setShown: (walletId: string) => {
+    window.localStorage.setItem(importedWalletDialogShown.getKey(walletId), 'true')
+  },
+  needShow: (walletId: string) => {
```

Comment:
The field `needShow` is confusing, what if using `setStatus(walletId: string, show: boolean)`